lms/djangoapps/support/static/support/jsx/logged_out_user.jsx

```diff
@@ -21,6 +21,9 @@ function LoggedOutUser({ platformName, loginQuery }) {
       <div className="row">
         <div className="col-sm-12">
           <a className="create-account" href={`/register${loginQuery}`}>{gettext(`Create an ${platformName} account`)}</a>
+          <p className="create-account-note">
+            {gettext(`If you are unable to access your account contact us via email using ${supportEmail}.`)}
```

Comment:
Haven't used `gettext` in this context, so apologies for the obvious question...but how will this work?

Will it try and translate the string 

```
"If you are unable to access your account contact us via email using ${supportEmail}."
```

or will it try and translate the string (I'm guessing this one?)

```
"If you are unable to access your account contact us via email using someSupportEmail@example.com"
```

Basically, is it more appropriate to do 

```javascript
{ `${gettext("If you are unable to access your account contact us via email using")} ${supportEmail}` }
```

Reply:
@jaebradley Yes it will translate it as `If you are unable to access your account contact us via email using someSupportEmail@example.com`.